ophyd/controls/mca.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import print_function
+import logging
+
+from collections import namedtuple
+
+from .signal import (EpicsSignal, EpicsSignalRO)
+from .device import OphydDevice
+from .device import (Component as C, DynamicDeviceComponent as DDC)
+
+logger = logging.getLogger(__name__)
+
+
+_roi_field_map = {'name': (EpicsSignal, '.R{n}NM'),
```

Comment:
so the `_Roi` has a deterministic order or `mca.roi1[0]` will be a different value each time.